src/containers/other/NFTView/elements/History/elements/HistoryItem.tsx

```diff
@@ -63,7 +63,7 @@ const HistoryItem = ({ item }: HistoryItemProps) => {
 						made an offer of{' '}
 						<b>
 							{Number(ethers.utils.formatEther(item.amount!)).toLocaleString()}{' '}
-							{paymentTokenInfo?.name}
+							{paymentTokenInfo?.symbol}
```

Comment:
<img width="899" alt="Screenshot 2022-07-14 at 22 40 47" src="https://user-images.githubusercontent.com/39112648/179090966-8eccc660-d4a6-4494-a78a-a515b2f3c608.png">

No symbol on mainnet, I was thinking mainnet contract addresses need to be added, but that wouldn't explain why the symbol is showing elsewhere

Reply:
Probably changes are not yet deployed to mainnet. If it is working fine on Rinkeby that should be good for now. I'll check with James though on this.